resources/enclosure.rb

```diff
@@ -32,3 +34,20 @@
 action :remove do
   remove
 end
+
+action :reconfigure do
+  item = load_resource
+  item.retrieve!
+  item.configuration
+end
+
+action :refresh do
```

Comment:
Cool. Yeah, sounds good. Agreed that notify is definitely the way to use these :)